README.rst

```diff
@@ -108,25 +108,19 @@ first name begins with 'D':
 
 .. code-block:: python
 
-    for user in UserModel.query("John", first_name__begins_with="D"):
+    for user in UserModel.query("Denver", UserModel.first_name.startswith("J"):
         print(user.first_name)
 
 Examples of ways to query your table with filter conditions:
 
 .. code-block:: python
 
-    for user in UserModel.query("John", filter_condition= (UserModel.email=="djohn@company.org")):
+    for user in UserModel.query("Denver", UserModel.email=="djohn@company.org"):
         print(user.first_name)
 
 .. code-block:: python
 
-    for user in UserModel.query("John", UserModel.email=="djohn@company.org"):
-        print(user.first_name)
-
-.. code-block:: python
-
-    # Deprecated, use UserModel.email=="djohn@company.org" instead
-    for user in UserModel.query("John", email__eq="djohn@company.org"):
+    for user in UserModel.query("Denver", UserModel.email=="djohn@company.org"):
```

Comment:
this is the same example as above (it was to show the difference between `email__eq ` and `UserModel.email==`)
so it is redundant.

Was there a specific reason to remove the second example?

Reply:
if we are just depracteing the old way we just should remove the second example altogether

Reply:
I didn't make this readme change :0

Reply:
You should look at the pr under pynamodb/Pynamodb instead of the one under Lyft (the Lyft repo isn't up to date with the latest pynamodb master, so a lot of these changes aren't actually mine)

Reply:
alright will review the other one